src/sign.py

```diff
@@ -103,6 +103,20 @@ def _arguments() -> argparse.Namespace:
         default=True,
         dest="use_ambient_credentials",
     )
+    sigstore.add_argument(
+        "--staging",
+        help="Use Sigstore's staging instances, instead of the default production instances",
+        action="store_true",
+        dest="sigstore_staging",
+    )
+    sigstore.add_argument(
+        "--identity-token",
+        help="the OIDC identity token to use",
+        required=False,
+        type=str,
+        default="",
```

Comment:
`sigstore-python` also supports `SIGSTORE_IDENTITY_TOKEN` and `SIGSTORE_STAGING` for these default values, if that's something we want to match.